### PR TITLE
404 fixes

### PR DESCRIPTION
**Pull request by divergentdave**
Here are a few 404 fixes, I found these languishing on a branch I had forgotten about.

### CODE REVIEW

File: inspectors/dhs.py [hunk 1]
```diff
@@ -86,7 +86,10 @@ def report_from(result, component, url):
   }
 
   link = result.select("td")[2].select("a")[0]
-  report_url = urllib.parse.urljoin(url, link['href'])
+  href = link['href'].replace("http://srvhq11c03-webs/", "/")
```
Comment (konklone) [inline inspectors/dhs.py]: 9_9

File: inspectors/epa.py [hunk 2]
```diff
@@ -108,6 +108,9 @@ def report_from(tds, published_on_dt, year):
   if report_url:
     report_url = urljoin(BASE_URL, report_url)
 
+  if report_url == "http://www.nsf.gov/oig/USMFS05.pdf":
+    report_url = "http://www.nsf.gov/oig/_pdf/USMFS05.pdf"
+
```
Comment (konklone) [inline inspectors/epa.py]: Looks like the NSF OIG asked nsf.gov to let HTTPS stay on for the OIG site, so these could be moved to `https://`.

**Comment by konklone** [comment]
"Oops I accidentally found some IG scraper fixes in the back of my closet"

You're a champ, @divergentdave. :)

File: inspectors/epa.py (hunk 2 above)
Comment (divergentdave) [inline inspectors/epa.py]: Fixed in e9f3509 on master

Comment (konklone) [inline inspectors/epa.py]: :+1: thanks!

**Pull request merged**